src/shared/components/ResourceEditor/editorUtils.ts

```diff
@@ -104,47 +137,47 @@ export async function resolveLinkInEditor({
         data = await nexus.Resource.list(undefined, undefined, {
           locate: url,
         });
+        if (
+          !data._total ||
+          (!data._total && url.startsWith('https://bbp.epfl.ch'))
```

Comment:
I think the repetition of `!data.total` before and after `||` is not needed (because if it was false, then it will be false after || check also). Maybe you want to check instead `!data.total || !url.startsWith('https://bbp.epfl.ch')`

As a separate issue, since we are only checking for the production domain, we will not be able to test this in other environments like dev, staging etc.

Reply:
yeah the `isExternalLink` will be extended to test dev/staging links @Dinika